moped-editor/src/views/projects/projectView/EditorTheme.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+// import {EditorThemeClasses} from 'lexical';
```

Comment:
This comment caught my eye on this review and would be great to clean up before merge. 🙏

Reply:
agreed!